sql/catalyst/src/main/scala/org/apache/spark/sql/util/ArrowUtils.scala

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package org.apache.spark.sql.execution.arrow
+package org.apache.spark.sql.util
```

Comment:
I don't want to add an `execution` package in catalyst, so I changed the pacakge for this internal class during the move.